document/js-api/index.bs

```diff
@@ -1121,8 +1121,7 @@ Note: ECMAScript doesn't specify any sort of behavior on out-of-memory condition
 
 The WebAssembly core specification allows an implementation to define limits on the syntactic structure of the module.
 While each embedding of WebAssembly may choose to define its own limits, for predictability the standard WebAssembly JavaScript Interface described in this document defines the following exact limits.
-An implementation must reject a module that exceeds these limits with a {{CompileError}}.
-In practice, an implementation may run out of resources for valid modules below these limits.
```

Comment:
Perhaps we should keep this sentence here as well, since it may apply even to static limits.